src/WellIO.jl

```diff
@@ -1,5 +1,20 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
+using Tables
 
+function load(filepath::String)
+    file = FileIO.query(filepath)
```

Comment:
Is this the new recommended method to use FileIO.jl? I thought we just needed to define a load with a File object, no need to call this `query` function. Can you please clarify?